README.md

```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
+
+The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using 4 options - [C# Markup](https://aka.platform.uno/csharp-markup) or [XAML]() for the UI and [MVUX](https://aka.platform.uno/mvux) or [MVVM]() for state management.
```

Comment:
Missing links for `[XAML]()` and `[MVVM]()`?

Reply:
For these links I would point directly to the variants in the workshop tutorial docs:

<img alt="image" src="https://github.com/unoplatform/Uno.Samples/assets/4793020/243e0d94-2ecc-4e42-b622-89833950ea4f">

I'm not sure we have aka links created for these though @agneszitte

Reply:
@vatsashah45, @kazo0, @nickrandolph
I think we could use this simple aka link (https://aka.platform.uno/simplecalc-workshop) that gives the overview of the four variants of the SimpleCalc workshop.

So we can mention something additional like:
`The sample app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).`